code/game/dna/genes/vg_powers.dm

```diff
@@ -240,7 +240,10 @@ Obviously, requires DNA2.
 
 /obj/effect/proc_holder/spell/targeted/remotetalk/cast(list/targets)
 	if(!ishuman(usr))	return
-	var/say = strip_html(input("What do you wish to say"))
+	var/say = input("What do you wish to say") as text|null
+	if(say == null || length(say) == 0)
```

Comment:
If this were any other language, I would think you've gone mad. I never imagined "" would == false